.github/config/assertoor/network_params_ethrex_only.yaml

```diff
@@ -0,0 +1,24 @@
+participants:
+  - el_type: ethrex
+    cl_type: lighthouse
+    cl_image: sigp/lighthouse:v7.0.0-beta.0
+    validator_count: 32
+    count: 3
+
+network_params:
+  # The address of the staking contract address on the Eth1 chain
+  deposit_contract_address: "0x4242424242424242424242424242424242424242"
+  electra_fork_epoch: 0 # This field won't be needed when ethereum-package is merged
+
+additional_services:
+  - assertoor
+  - dora
+
+assertoor_params:
+  run_stability_check: false
+  run_block_proposal_check: false
```

Comment:
why are you disabling these two checks?

Reply:
But the fact that it's only ethrex clients doesn't possibly change the scope of the test? Im asking, not sure of the answer. Were you able to run them to make sure they pass?